src/tds/read.c

```diff
@@ -240,8 +240,12 @@ tds_get_n(TDSSOCKET * tds, void *dest, size_t need)
 			dest = (char *) dest + have;
 		}
 		need -= have;
-		if (TDS_UNLIKELY(tds_read_packet(tds) < 0))
+		if (TDS_UNLIKELY(tds->recv_packet->capacity < 2
```

Comment:
`capacity` can't be so small

Reply:
Fair point; I just wanted to preclude any possibility that the following array access would be invalid.